docs/api-reference-1/frontity.md

```diff
@@ -258,7 +321,10 @@ const Component = () => (
 It's a React component that creates global styles for the whole Frontity site.
 
 {% hint style="warning" %}
-**Using `<Global>` for other than html tags is not recommended** because Frontity is not able to optimize it. That means you can use it for tags like `html`, `body` , `a`, `img`, and so on... but **avoid it for classes**. Use either the css prop or styled components instead.
+**Using `<Global>` for other than HTML tags is not recommended** because Frontity is not able to optimize it.
+That means you can use it for tags like `html`, `body` , `a`, `img`, and so on...
+But **avoid it for classes**.
+Use either the CSS prop or styled-components instead.
```

Comment:
We used `styled components` when referring to components styled with `styled` instead of `styled-components` to avoid confusion with the [`styled-components`](https://styled-components.com/) library, because we don't use that, we use Emotion. So we felt like `styled components` was more generic, and `styled-components` more specific to the library.

Anyway, up to you guys now. I just wanted to tell you the reason 🙂